library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/GlobalBounds.kt

```diff
@@ -6,7 +6,17 @@
 
 package com.datadog.android.sessionreplay.internal.recorder
 
-internal data class GlobalBounds(
+/**
+ * Defines the dimension and positions in Global coordinates for a geometry.
```

Comment:
imo we need to say what is the meaning of global here, like (0,0) is the corner of the screen, not app decor view, etc.